dagster/terraform/azure/main.tf

```diff
@@ -8,3 +8,26 @@ resource "kubernetes_namespace" "dagster" {
     }
   }
 }
+
```

Comment:
Can you rename this TF module from `azure` to `minio`, this way it can also be used by other providers in the future without it being confusing or duplicating the code.

Reply:
I think that might rip out some existing installations unfortunately